### PR TITLE
chore: remove kubernetes scripts from repo

### LINKED ISSUE #193: [TASK]: remove kubernetes scripts from repo
**Describe the task**
the kubernetes scripts and keycloak theme has been move to the orchestration repo. These can now be removed

**Task Items**
- [ ] remove the kubernetes and keycloak-theme directories from the repo

**Goal**
A clear and concise description of what the end result must have for it to be approved as complete.

**Screenshots**
If applicable, add screenshots or designs

**Additional context**
Add any other context about the task here

### PR DESCRIPTION
**Pull request by dgauldie**
# Description

remove the kubernetes and keycloak-thems folders from the repo. These have been moved to the orchestration repo

Resolves #193 

## Type of change

- [x] Repo organization
- [x] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide reproduction instructions and list any relevant details for your test configuration

- [ ] All build scripts and documentation are updated

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Review by YohannParis — DISMISSED** [review]

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -1,59 +1,79 @@
...
 
-1. Stand up the Gatweay servers by running the `gateway.sh up` script from within the `kubernetes/local` directory.
-2. Start up the Backend Quarkus server using `./gradlew quarkusDev` or `quarkus dev` command
-3. Start the front end by doing `yarn install` followed by `yarn workspaces webapp dev`
-4. Navigate your browser to `localhost:8078` 
+1. Stand up the TERArium services by running the `deploy-terarium.sh up` script from within the
+	 [Orchestration](https://github.com/DARPA-ASKEM/orchestration) repository
+2. Stop the `hmi-server` and `webapp` pods
+3. Start up the Backend Quarkus server using `./gradlew quarkusDev` or `quarkus dev` command
+4. Start the front end by doing `yarn install` followed by `yarn workspaces webapp dev`
```
Comment (YohannParis) [inline .github/CONTRIBUTING.md:17]:
```suggestion
4. Start the front end by doing `yarn install` followed by `yarn dev`
```

File: .github/CONTRIBUTING.md [hunk 2]
```diff
@@ -1,59 +1,79 @@
...
 
 To develop and test the core application:
 
-1. Stand up the Gatweay servers by running the `gateway.sh up` script from within the `kubernetes/local` directory.
-2. Start up the Backend Quarkus server using `./gradlew quarkusDev` or `quarkus dev` command
-3. Start the front end by doing `yarn install` followed by `yarn workspaces webapp dev`
-4. Navigate your browser to `localhost:8078` 
+1. Stand up the TERArium services by running the `deploy-terarium.sh up` script from within the
+	 [Orchestration](https://github.com/DARPA-ASKEM/orchestration) repository
+2. Stop the `hmi-server` and `webapp` pods
```
Comment (YohannParis) [inline .github/CONTRIBUTING.md:15]: You should add the command to do so.

Comment (YohannParis) [reply .github/CONTRIBUTING.md:15]: And edit the local deployment to have a replica of `0` to easily kill them

File: .github/CONTRIBUTING.md (hunk 1 above)
Comment (dgauldie) [reply .github/CONTRIBUTING.md]: fixed

File: .github/CONTRIBUTING.md (hunk 2 above)
Comment (dgauldie) [reply .github/CONTRIBUTING.md]: fixed

**Review by YohannParis — APPROVED** [review]

**Pull request merged**